subiquity/server/controllers/filesystem.py

```diff
@@ -616,6 +616,11 @@ def guided_zfs(self, gap, choice: GuidedChoiceV2):
         rpool.create_zfs(f"ROOT/ubuntu_{uuid}/var/spool")
         rpool.create_zfs(f"ROOT/ubuntu_{uuid}/var/www")
 
+        userdata_uuid = gen_zsys_uuid()
+        rpool.create_zfs("USERDATA", canmount="off", mountpoint="/")
```

Comment:
Do we have to specify the mountpoint here? For "ROOT" we use the string "none" and `create_zfs` has `mountpoint=None` by default.

Reply:
Works for me.  I think it doesn't matter with `canmount="off"`  but consistency sounds good.  Fixed and vm tested!

Reply:
Thanks for checking!